test/integration/local_transactions_test.rb

```diff
@@ -365,7 +367,7 @@ class LocalTransactionsTest < ActionDispatch::IntegrationTest
     end
 
     should "show advisory message that we have no url" do
-      assert page.has_content?("We don't have a link for their website.")
+      assert page.has_content?("We don't have a link for their website")
```

Comment:
Do we need to drop this period? I wasn't sure if it was actually a mistake.

Reply:
I am not 100% sure but the [current results page](https://www.gov.uk/report-stray-dog/milton-keynes) doesn't have one, so I took it out.

Reply:
I can't see that text on the example you linked (which means I'm pleased Milton Keynes does have a service for reporting stray dogs)

I can see the period in the source code though: https://github.com/alphagov/frontend/blob/a61c3bd90c60e11d50228b6b6983ed8b82746a00/app/views/local_transaction/results.html.erb#L49 unless we're somehow talking about different things?